test/contracts-network/colony-network-extensions.js

```diff
@@ -278,4 +285,43 @@ contract("Colony Network Extensions", (accounts) => {
       await checkErrorRevert(colony.uninstallExtension(TEST_EXTENSION, { from: ROOT }), "colony-network-extension-not-installed");
     });
   });
+
+  describe("using extensions", () => {
+    it("allows network-managed extensions to lock and unlock tokens", async () => {
+      const TEST_VOTING_TOKEN = soliditySha3("VotingToken");
+
+      const tokenLockingAddress = await colonyNetwork.getTokenLocking();
+      const tokenLocking = await ITokenLocking.at(tokenLockingAddress);
+
+      const testVotingToken = await TestVotingToken.new();
+      const resolver = await Resolver.new();
+      await setupEtherRouter("TestVotingToken", { TestVotingToken: testVotingToken.address }, resolver);
+      await metaColony.addExtensionToNetwork(TEST_VOTING_TOKEN, resolver.address);
+
+      await colony.installExtension(TEST_VOTING_TOKEN, 1, { from: ROOT });
+      const extensionAddress = await colonyNetwork.getExtensionInstallation(TEST_VOTING_TOKEN, colony.address);
+      const extension = await TestVotingToken.at(extensionAddress);
+
+      const lockCountPre = await tokenLocking.getTotalLockCount(token.address);
+
+      await extension.lockToken();
+
+      const lockCountPost = await tokenLocking.getTotalLockCount(token.address);
+      expect(lockCountPost.sub(lockCountPre)).to.eq.BN(1);
+
+      await extension.unlockTokenForUser(ROOT, lockCountPost);
```

Comment:
Much like in the reputation emission PR, it'd be good to see the limits of the underlying function are still respected (i.e. cannot increment more than 1).

Should this test be broken up in to multiple tests?